src/locales/vocabulary.js

```diff
@@ -99,47 +100,42 @@ export default {
   'editdialog.anonymousMessage2_login': 'log in to OpenStreetMap',
   'editdialog.anonymousMessage3': ', your changes will be immediate.',
   'editdialog.add_major_tag': 'Add',
-  'editdialog.location_checkbox': 'Change location',
+  'editdialog.location_checkbox': 'New location',
   'editdialog.location_placeholder': 'eg. across the street',
   'editdialog.location_editor_to_be_added': 'The position cannot be edited here yet, you can do so in the <a href="__link__">iD editor</a>.',
-  'editdialog.place_cancelled': 'Place permanently closed (delete)',
+  'editdialog.place_cancelled': 'Permanently closed (delete)',
   'editdialog.comment': 'Comment (optional)',
-  'editdialog.comment_placeholder': 'link to the source of information etc.',
+  'editdialog.comment_placeholder': 'Extra details, link to source, etc.',
```

Comment:
It could make sense for a "note dialog" to encourge users to fill extra details, but would prefer if they did not waste time on filling additonal details which probably cant be saved in OSM db anyways..

For Logged-in variant it is definitely wrong, the comment should not contain any details about the item. Only about the source.

If you still think its good, we could split the message in two. 🙂

Reply:
I didn't intend to infer that more information for the object should be added.

Maybe:
> Comment, link to source, etc.

Reply:
Yes, I am ok with "Comment, link to source, etc." though the label for the field is "Comment (optional)" already, so its probably redundant. I am ok with both, feel free to edit it as you wish 🙂 


Another idea -  we can add eg. to the placeholder "eg. link to source, etc."


<img width="688" alt="Screenshot 2023-01-22 at 17 07 18" src="https://user-images.githubusercontent.com/385047/213926186-04e1b856-20f2-4842-a8aa-b1243d2ee95b.png">